src/skin/skinloader.cpp

```diff
@@ -71,8 +71,6 @@ QString SkinLoader::getConfiguredSkinPath() const {
         if (configSkin.isEmpty()) {
             configSkin = getDefaultSkinName();
         }
-        m_pConfig->set(ConfigKey("[Config]", "ResizableSkin"),
-                       ConfigValue(configSkin));
```

Comment:
It should not be possible to set configuration values outside of the preferences code! The whole preferences system could really use a thorough cleanup. It's really brittle.